docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
-
-   Then grab your Cloudserver’s logs with the command:
-
-   ::
-
-     $ kubectl logs my-zenko-cloudserver-<id> | grep 'Instance ID'
-
-
-   Using the present sample values, this command returns:
-
-   ::
-
-     $ kubectl logs my-zenko-cloudserver-76f657695-j25wq | grep 'Instance ID'
-
-     {"name":"S3","time":1532632170292,"req_id":"effb63b7e94aa902711d",\
-     "level":"info","message":"this deployment's Instance ID is \
-     7586e994-01f3-4b41-b223-beb4bcf6fff6","hostname":"my-zenko-cloudserver-\
-     76f657695-j25wq","pid":19}
-
-   Copy the instance ID.
-
-6. Open https://admin.zenko.io/user in a web browser. You may be prompted to
-   authenticate through Google.
-
-7. Click the button to **Register My Instance**.
-
-8. Paste the instance ID into the Instance ID dialog. Name the instance what
-   you will.
-
-Your instance is registered.
+.. spelling::
+
+   Quickstart
+   subdirectory
+   Todo
+
+Installation Guide
+++++++++++++++++++
+
+While it is possible to run Zenko on a single machine, it's designed for
+cluster operation. If you can set up a Kubernetes cluster on your own, review
+the **General Cluster Requirements** and skip to **Installing Zenko**, below.
+
+Otherwise, you can set up a cluster quickly using MetalK8s_, Scality's
+open-source Kubernetes cluster project, as described in "Setting Up a Metal K8s
+Cluster," below.
+
+The following section describes general cluster requirements, which are tested
+on Metal K8s. Because MetalK8s is designed to operate without support from
+public cloud resources, the following sizing requirements are assumed good for
+other cloud Kubernetes deployments, where such resources are preinstalled and
+available on demand.
+
+####################
+Setting Up a Cluster
+####################
+
+****************************
+General Cluster Requirements
+****************************
+
+Setting up a testing cluster requires at least three machines (these can be
+VMs) running CentOS_ 7.4 (The recommended mimimum for Zenko production service
+is five server nodes with three masters/etcds, but for testing and
+familiarization, three masters and three nodes is fine). You must have SSH
+access to these machines and they must have SSH access to each other. (You
+can copy SSH credentials from one machine to the next and log in once to
+ensure each machine has been added to the others' recognized hosts lists).
+Each machine acting as a Kubernetes_ node must also have at least one disk
+available to provision storage volumes.
+
+
+.. _MetalK8s: https://github.com/scality/metal-k8s/
+.. _CentOS: https://www.centos.org
+.. _Kubernetes: https://kubernetes.io
+
+
+Sizing
+======
+
+Sizing for Metal K8s
+--------------------
+
+-  Cores
+
+   -  2 for etcd
+   -  4 for master
+   -  4 for node
+
+-  RAM
+
+   -  4 GB for etcd
+   -  8 GB for master
+   -  8 GB for node
+
+-  Storage (for the system)
+
+   -  20 GB for the root filesystem
+   -  16 GB for etcd

```

Comment:
If we size `etcd` storage, we should document how to set up the system(s) to have `etcd` storage not on `/` (which is good practice in any case).

Reply:
Is this a show-stopper? I need to know either: a) that I can omit the etcd sizing requirement or b) how to configure the system to put etcd somewhere other than / .

If I don't hear back, I'm filing this under "nice to have" and moving the other changes along.

Reply:
I think this ought to be discussed with out TS folks. Right now, we don't have any recommendations for putting various types of data on different volumes, but maybe that should be the case.